application/core/plugins/ReactEditor/css/editorModal.css

```
@@ -1,42 +1,215 @@
 
 #activate_editor .mb-16 {
-    margin-bottom: 16px;
+  margin-bottom: 16px;
 }
 
 .hint-text-box {
-    background-color: #F9F9FB; /* $g-100*/
-    border-radius: 3px;
-    padding-left: 16px;
-    padding-right: 16px;
+  background-color: #f9f9fb; /* $g-100*/
```

Stylelint (17.12.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)